File: client/src/main/java/com/jraska/github/client/logging/FirebaseCrashProxy.java
package com.jraska.github.client.logging;

import com.google.firebase.crash.FirebaseCrash;

class FirebaseCrashProxy {
  void log(String message) {
    FirebaseCrash.log(message);
  }

  void report(Throwable error) {
    FirebaseCrash.report(error);
  }
}

File: client/src/main/java/com/jraska/github/client/logging/FirebaseCrashProxy.kt
package com.jraska.github.client.logging

import com.google.firebase.crash.FirebaseCrash

internal open class FirebaseCrashProxy {
  open fun log(message: String) {
    FirebaseCrash.log(message)
  }

  open fun report(error: Throwable?) {
    FirebaseCrash.report(error)
  }
}
